myproject/src/index.js

```diff
@@ -1,8 +1,29 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import CreateNote from "./components/CreateNote";
+import { BrowserRouter as Router, Route, NavLink } from "react-router-dom";
 
-ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+ReactDOM.render(
+  <Router>
+    <div className="App">
```

Comment:
Remove all this jsx view stuff from your index.js.. Factor it out somewhere else.